temporal/t.rast3d.mapcalc/t.rast3d.mapcalc.html

```diff
@@ -19,8 +19,3 @@ <h2>SEE ALSO</h2>
 <h2>AUTHOR</h2>
 
 S&ouml;ren Gebbert, Th&uuml;nen Institute of Climate-Smart Agriculture
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 16 `<a href="r3.mapcalc.html">r3.mapcalc</a>,` should be `<a href="r3.mapcalc.html">r3.mapcalc</a>`.